core/src/main/java/io/grpc/internal/PickFirstLeafLoadBalancer.java

```diff
@@ -324,6 +432,9 @@ private Subchannel createNewSubchannel(SocketAddress addr) {
             new EquivalentAddressGroup(addr)))
         .addOption(HEALTH_CONSUMER_LISTENER_ARG_KEY, hcListener)
             .build());
+    if (subchannel == null) {
+      throw new IllegalArgumentException("Was not able to create subchannel for " + addr);
```

Comment:
IIRC LB should never throw. Otherwise it cause channel panic.

Reply:
I can return a null, and then in requestConnection try the next address.